UIKitTemplate/UIKitTemplate/CustomUIButton.swift

```diff
@@ -0,0 +1,36 @@
+// CustomUIButton.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Кастомная кнопка для главного экрана и экрана выбора прожарки зерен
```

Comment:
```suggestion
/// Кастомная кнопка для главного экрана и экрана выбора прожарки зерен
```
если это универсальная кнопка ее можно везде юзать поправь описание